dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server

```diff
@@ -884,27 +884,28 @@ findRealPath() {
 
 mergeJVMOptions()
 {
-    jvmOptions=$1
+  jvmOptions=$1
 
-    if [ -f "$jvmOptions" ]; then
-      READ_ETC=0
-      saveIFS=$IFS
-      IFS=$newline
-
-      for option in `readNativeFile "$jvmOptions" '[#-]' | LC_ALL=C tr -d '\r'`; do
-        if [ -n "$option" ]; then
-          case $option in
-          \#*);;
+  if [ -f "$jvmOptions" ]; then
+    READ_ETC=0
+    saveIFS=$IFS
+    IFS=$newline
+    
+    for option in $(readNativeFile "$jvmOptions" '[#-]' | LC_ALL=C tr -d '\r'); do
+      if [ -n "$option" ]; then
+        case $option in
+          \#*) 
+              ;;
           *)
-            escapeForEval "${option}"
-            SERVER_JVM_OPTIONS_QUOTED="${SERVER_JVM_OPTIONS_QUOTED} '${option}'"
-            ;;
-          esac
-        fi
-      done
-      IFS=$saveIFS
-    fi
-
+              escapeForEval "${option}"
```

Comment:
escapeForEval doesn't return anything.   The result needs to be assigned separately.  
escapeForEval also surrounds the result in quotes.  So adding another set of quotes around the result becomes unnecessary.